packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -141,6 +151,101 @@ const getDocs: () => JSX.Element = () => {
         <LocalCameraSwitcherExample />
       </Canvas>
 
+      <Heading>Pinning Participants</Heading>
+      <DetailedBetaBanner />
+      <Description>
+        The VideoGallery provides a contextual menu for each remote video tile by clicking menu button next to the
+        display name. The contextual menu will have a menu item to pin a participant's video tile such that only pinned
+        participants are shown in the GridLayout. This is shown in the two screenshots below. You can also try it out in
+        any of the above VideoGallery components on this Docs page. Pinned participants will be shown in the order that
+        they are pinned.
+      </Description>
+      <Stack horizontal={true} tokens={{ childrenGap: '0.5rem' }}>
+        <Image
+          style={{ width: '100%', height: 'auto' }}
+          src="images/pin-menu-item-video-gallery.png"
+          alt="Pin menu item in VideoGallery"
+        />
+        <Image
+          style={{ width: '100%', height: 'auto' }}
+          src="images/pinned-video-gallery.png"
+          alt="Pinned participant in VideoGallery"
+        />
+      </Stack>
+      <Description>
+        When screensharing is active, pinned participants are shown in the horizontal gallery as shown in the two
+        screenshots below.
+      </Description>
+      <Stack horizontal={true} tokens={{ childrenGap: '0.5rem' }}>
+        <Image
+          style={{ width: '100%', height: 'auto' }}
+          src="images/pin-menu-item-video-gallery-with-screenshare.png"
+          alt="Pin menu item in VideoGallery with screenshare active"
+        />
+        <Image
+          style={{ width: '100%', height: 'auto' }}
+          src="images/pinned-video-gallery-with-screenshare.png"
+          alt="Pinned participant in VideoGallery with screenshare active"
+        />
+      </Stack>
+      <Description>
+        Pinned participants can be unpinned through the contextual menu as shown in the two screenshots below.
+      </Description>
+      <Stack horizontal={true} tokens={{ childrenGap: '0.5rem' }}>
+        <Image
```

Comment:
Overall the docs read great and look fantastic. something I am wondering though is can we make this 1 image here somehow? is there a state that might showcase what these two image are showing in one image? not blocking though!

Reply:
I will try a GIF :)

Reply:
Changed screenshots to short video clips